doc/start/os-recommendations.rst

```diff
@@ -38,19 +38,27 @@ Platforms
 
 The charts below show how Ceph's requirements map onto various Linux
 platforms.  Generally speaking, there is very little dependence on
-specific distributions aside from the kernel and system initialization
+specific distributions outside of the kernel and system initialization
 package (i.e., sysvinit, systemd).
 
-+--------------+--------+--------+---------------+----------+--------+
-| Release Name | Tag    | CentOS | Ubuntu        | OpenSUSE | Debian |
-+==============+========+========+===============+==========+========+
-| Quincy       | 17.2.z | 8      | 20.04         | 15.3     | 11     |
-+--------------+--------+--------+---------------+----------+--------+
-| Pacific      | 16.2.z | 8      | 18.04, 20.04  | 15.2     | 10, 11 |
-+--------------+--------+--------+---------------+----------+--------+
-| Octopus      | 15.2.z | 7, 8   | 18.04, 20.04  | 15.2     | 10     |
-+--------------+--------+--------+---------------+----------+--------+
-
++--------------+--------+------------------------+---------------+----------+-----------------+
+| Release Name | Tag    | CentOS                 | Ubuntu        | OpenSUSE | Debian          |
++==============+========+========================+===============+==========+=================+
+| Quincy       | 17.2.z | 8 :sup:`A`             | 20.04         | 15.3     | 11 :sup:`C`     |
++--------------+--------+------------------------+---------------+----------+-----------------+
+| Pacific      | 16.2.z | 8 :sup:`A`             | 18.04, 20.04  | 15.2     | 10 :sup:`C` 11  |
++--------------+--------+------------------------+---------------+----------+-----------------+
+| Octopus      | 15.2.z | 7 :sup:`B` 8 :sup:`A`  | 18.04, 20.04  | 15.2     | 10 :sup:`C`     |
++--------------+--------+------------------------+---------------+----------+-----------------+
+
+- **A**: Ceph provides packages and has done comprehensive tests on the software in them.
+- **B**: Ceph provides packages and has done basic tests on the software in them.
+- **C**: Ceph provides packages only. No tests have been done on these releases.
+
+.. note::
+   **For Centos 7 Users** 
+   
+   ``Btrfs`` is no longer tested on Centos 7 in the Octopus release. We recommend using ``bluestore`` instead.
 
 Notes
```

Comment:
looks like we can get rid of the rest of this notes section now as well

Reply:
Yeah, that's the next commit that waits on this one to merge.